src/components/Footer/Footer.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import Link from '../link/link';
-import Container from '../container/container';
+import Link from '../Link/Link';
+import Container from '../Container/Container';
```

Comment:
Now that each file is named similarly just with a different extension, is there a potential for conflict here? Seeing as the site works, I'm guessing webpack tries `.jsx` files before `.scss` but isn't it somewhat ambiguous to which module this `import` goes? I do see that you did update all the styling `import`s to hit the `.scss` file directly. If this is common practice then I'm fine with it, just curious.